components/infobox/wikis/counterstrike/infobox_league_custom.lua

```diff
@@ -202,9 +206,8 @@ function CustomLeague:_createTier(args)
 	end
 
 	content = content .. '[[Category:' .. tierDisplay.. ' Tournaments]]'
```

Comment:
Is this needed still?

Reply:
since the categories are set in the function that this function overwrites yes they are needed unttil CS can be switched to the commons function